js/src/block-editor/controls/file.js

```diff
@@ -0,0 +1,121 @@
+/**
```

Comment:
This file is mainly copied from `image.js`, so they could probably use a common component that they both call.

Reply:
Actually, there are enough differences that a common component might make it more complex.